sklearn/tests/test_array_out.py

```diff
@@ -0,0 +1,27 @@
+import pytest
```

Comment:
The name of this file is weird, probably because this has been backported from the other PR?